concepts/05 UI Components/List/00 Getting Started with List/05 Group Items.md

```diff
@@ -0,0 +1,20 @@
+The **List** can display items as a two-level hierarchy: parent and child items. To group items, set the [grouped](/Documentation/ApiReference/UI_Components/dxList/Configuration/#grouped) property to **true** and ensure that data is structured as a [hierarchical array of objects](/Documentation/Guide/UI_Components/List/Grouping/In_the_Data_Source/). Each object must contain a **key** property (parent) and an **items** property (child items).
```

Comment:
It feels like we should point the reader more obviously to the article with more information.

```suggestion
The **List** can display items as a two-level hierarchy: parent and child items. To group items, set the [grouped](/Documentation/ApiReference/UI_Components/dxList/Configuration/#grouped) property to **true** and ensure that data is structured as a hierarchical array of objects. Each object must contain a **key** property (parent) and an **items** property (child items). Refer to the following help topic for more information: [Grouping in the Data Source](/Documentation/Guide/UI_Components/List/Grouping/In_the_Data_Source/).
```